docs/src/main/asciidoc/_developer-guide/Embedded_API.adoc

```diff
@@ -0,0 +1,110 @@
+[[embedded-api]]
+= Embedded API
+:author:            James R. Perkins
+:email:             jperkins@redhat.com
+:idprefix:
+:idseparator:       -
+
+include::../common_attributes.adoc[]
+
+The embedded API can be used to launch {appservername} within a currently running process.
+
+The embedded server can be reinitialized with a different JBoss Home. However the module directory, `module.path`
+system property, and the modules system packages, `jboss.modules.system.pkgs` system property, are effectively static.
+This means that creating a new embedded server or host controller within the same VM will not allow overriding the
+modules directory or the system packages.
+
+By default the logging subsystem is not used to configure logging for the embedded server. A hint can be used on the
+configuration, `Configuration.setEmbeddedLoggingHint(false)`, which allows the logging subsystem to configure logging.
+The system property `java.util.logging.manager` must be set to `org.jboss.logmanager.LogManager` before any attempt is
+made to create a JUL logger.
```

Comment:
I'm not clear on how this last sentence relates to the rest of the paragraph. Does it only apply when setEmbeddedLoggingHint(false) is used?

Reply:
Yes. I'll see if I can reword it or just remove it. It was more of a warning that it will blow up if that value is `false`.